api/envoy/data/accesslog/v2/accesslog.proto

```diff
@@ -70,7 +70,7 @@ message ConnectionProperties {
 }
 
 // Defines fields that are shared by all Envoy access logs.
-// [#next-free-field: 22]
+// [#next-free-field: 23]
```

Comment:
This is v2 API and frozen. Do you want to be updating v3?

Reply:
Thanks! Yeah, V3 should be used.